TensorMath.lua

```diff
@@ -388,6 +388,18 @@ wrap("add",
         {name=real, default=1},
         {name=Tensor}})
 
+
+wrap("csub",
+     cname("sub"),
```

Comment:
This conflicts with the [sub](https://github.com/torch/torch7/blob/master/doc/tensor.md#tensor-subdim1s-dim1e---dim4s--dim4e) function, right ?